translator/trace/zipkinv1/thrift.go

```diff
@@ -24,12 +24,32 @@ import (
 	"net"
 
 	tracepb "github.com/census-instrumentation/opencensus-proto/gen-go/trace/v1"
+	jaegerzipkin "github.com/jaegertracing/jaeger/model/converter/thrift/zipkin"
 	"github.com/jaegertracing/jaeger/thrift-gen/zipkincore"
 	"google.golang.org/protobuf/types/known/timestamppb"
 
-	idutils "go.opentelemetry.io/collector/internal/idutils"
+	"go.opentelemetry.io/collector/internal/idutils"
+	"go.opentelemetry.io/collector/internal/model"
 )
 
+var _ model.TracesDecoder = (*thriftDecoder)(nil)
+
+type thriftDecoder struct{}
+
+// DecodeTraces from Thrift bytes.
+func (t thriftDecoder) DecodeTraces(buf []byte) (interface{}, error) {
+	spans, err := jaegerzipkin.DeserializeThrift(buf)
+	if err != nil {
+		return nil, err
+	}
+	return v1ThriftBatchToOCProto(spans)
+}
+
+// NewThriftUnmarshaler returns an unmarshaler for Zipkin Thrift.
+func NewThriftUnmarshaler() model.TracesUnmarshaler {
+	return model.NewTracesUnmarshaler(thriftDecoder{}, toTranslator{})
+}
```

Comment:
Probably zipkin will never support other signals than Traces but should we keep consistency and call it `NewThriftTracesUnmarshaler`?